rllib/core/models/tf/primitives.py

```diff
@@ -42,8 +42,8 @@ def __init__(self, config: ModelConfig):
         if not is_output_decorated(self.__call__):
             _raise_not_decorated_exception("output")
 
-    @check_input_specs("input_spec", cache=True)
-    @check_output_specs("output_spec", cache=True)
```

Comment:
Turning off caching here helped me discover an error. We can switch it on by default when RLModules have stabilized but I don't think there is a need for caching now but rather for catching some bugs 🙂